Assets/HoloToolkit/Input/Tests/Scripts/SphereKeywords.cs

```diff
@@ -1,13 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using HoloToolkit.Unity;
 using UnityEngine;
 
-public class SphereKeywords : MonoBehaviour
+public class SphereKeywords : Interactable
 {
-    public void ChangeColor(string color)
+    protected override void KeywordRecognized(string keyword)
     {
-        switch(color.ToLower())
+        switch(keyword.ToLower())
```

Comment:
We'll need to eventually rethink this design and remove the switch and .ToLower() use due to the limiting ordinal comparison going on behind the scenes (diacritics come to mind) and the tiny perf hit. OK for now.